pagination.go

```diff
@@ -28,13 +34,19 @@ func (p Paged) CurrentPage() int64 {
 	return p.Page
 }
 
+func (p Paged) NextCursor() string { return p.Cursor }
+
+func (p Paged) HasNextCursor() bool { return p.Cursor != "" }
```

Comment:
🟢 Maybe just `HasNextPage`? Usually I see this wording, however up to you what to use.